docs/standard/native-interop/charset.md

```diff
@@ -12,10 +12,10 @@ The way `char` values, `string` objects, and `System.Text.StringBuilder` objects
 
 The following table shows a mapping between each charset and how a character or string is represented when marshalled with that charset:
 
-| CharSet | Windows            | Unix on .NET Core 2.2 and earlier | Mono and Unix on .NET Core 3.0 and later |
-|---------|--------------------|-----------------------------|------------------------------------------|
-| Ansi    | `char` (ANSI)      | `char` (UTF-8)              | `char` (UTF-8)                           |
-| Unicode | `wchar_t` (UTF-16) | `char16_t` (UTF-16)         | `char16_t` (UTF-16)                      |
-| Auto    | `wchar_t` (UTF-16) | `char16_t` (UTF-16)         | `char` (UTF-8)                           |
+| `CharSet` | Windows            | .NET Core 2.2 and earlier on Unix | .NET Core 3.0 and later and Mono on Unix |
```

Comment:
I think that the second column should read "Mono and .NET Core 3.0 and later on Unix"; Mono doesn't exclusively run on Unix systems.

Reply:
I don't think Mono on Windows behaves the same as Mono on Unix.

/cc @jkoritzinsky

Reply:
Mono follows .NET Framework and .NET Core on Windows. That’s why I named that column “Windows”. If you have a clearer way of phrasing it, I’m all for it.

Reply:
So the last column applies to .NET Core 3.0 and later on Unix, and Mono on Unix. Mono should be put after .NET Core 3.0, otherwise it seems to be Mono 3.0 and .NET Core 3.0?

Reply:
Sounds good to me!